client/analytics/settings/historical-data/style.scss

```diff
@@ -112,4 +112,7 @@
 .woocommerce-settings-historical-data__actions {
 	align-items: center;
 	display: flex;
+	button:disabled {
+		opacity: 0.5 !important;
+	}
```

Comment:
This works great, but I think should be removed because its a problem across all wc-admin. Issue to track here: https://github.com/woocommerce/woocommerce-admin/issues/5046

Reply:
Thank you @psealock, got it!
This was removed in the commit b481d41